news/schema.rst

```diff
@@ -0,0 +1,22 @@
+**Added:**
+
+* Schemas and exemplars for the collections.
+Database entries are checked against the schema, making sure that all the
+required fields are filled and the values are the same type(s) listed in the
+schema. The schema also includes descriptions of the data to be included.
+The exemplars are examples which have all the specified fields and are
+used to check the validation.
```

Comment:
This is not valid rst, which requires the paragraph to be indented after the bullet point.

Reply:
I'll merge this and fix it though.